src/components/PageContent/StepIndicator/StepIndicator.jsx

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import classNames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import * as Rivet from "../../util/Rivet";
+import { TestUtils } from "../../util/TestUtils";
+import Step from "./Step"
+
+const testIds = TestUtils.StepIndicator
+/**
+ * Show the user's position in and progress through a multi-step process
+ */
+const StepIndicator = ({
```

Comment:
Thoughts on calling this Steps instead of StepIndicator? Or I suppose StepsIndicator but personally I like Steps more...

The more I look at it, it also seems to me that the indicator is the nested element... anyway, my vote is Steps/Step. :)

Reply:
I originally debated on the naming too but StepIndicator keeps the primary component inline and connected to the rivet component. While the Steps matches the css of the container component the two components are not really for independent use.  The name StepIndicator does describe the combined functionality and even if the Step components are switch out for custom the container is still the start of the Indicator.